aspnet/mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/adding-a-model.md

```diff
@@ -2,7 +2,7 @@
 uid: mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/adding-a-model
 title: "Adding a Model (C#) | Microsoft Docs"
 author: Rick-Anderson
-description: "Note: An updated version of this tutorial is available here that uses ASP.NET MVC 5 and Visual Studio 2013. It's more secure, much simpler to follow and demo..."
+description: This tutorial will teach you the basics of building an ASP.NET MVC Web application using Microsoft Visual Web Developer 2010 Express Service Pack 1. You will learn how to add a model and model class and also work with SQL server compact to create a connection string."
```

Comment:
This gets it down to 144 characters: "Learn to build an ASP.NET MVC Web application using Microsoft Visual Web Developer 2010 Express Service Pack 1. Work with SQL server compact to create a connection string."